packages/fakta-fordel-beregningsgrunnlag/src/components/fordeling/FordelPeriodeFieldArray.tsx

```diff
@@ -445,15 +444,12 @@ export const FordelPeriodeFieldArray = ({
                   />
                 </Table.DataCell>
                 <Table.DataCell align="right" textSize="small">
-                  {skalViseSletteknapp(index, fields, skalIkkeEndres) && (
-                    <Button
-                      size="small"
-                      icon={<XMarkIcon aria-hidden color="var(--ax-danger-600)" />}
-                      onClick={() => remove(index)}
-                      type="button"
-                      variant="tertiary-neutral"
-                    />
-                  )}
+                  <RhfFieldArray.RemoveButton
```

Comment:
er det sånn å forstå at du har lagt opp til å bruka remove og appendbutton utanfor ein RhfFieldArray? Eg syns det var litt forvirrande. Eg har ikkje eksponert mange komponentar på denne måten, men ser ein til dømes på akselkode, så blir sånne komponentar brukt i kontekst av hovedkomponenten. For meg er iallfall det mest logisk og derfor vart eg litt forvirra av dette. Så mogleg ein burde fjerna den sterke koblinga og flytta desse to komponentane ut

Reply:
Godt poeng. Jeg endrer dem til  `RhfFieldArrayRemoveButton` og `RhfFieldArrayAppendButton`